deploy/template.yaml

```diff
@@ -162,6 +162,28 @@ Resources:
             Path: /stub-dcs-check-passport
             Method: POST
 
+  JwtVerification:
+    Type: AWS::Serverless::Function
+    Properties:
+      FunctionName: !Sub "ipv-shared-attributes-${Environment}"
+      Handler: uk.gov.di.ipv.cri.passport.sharedattributes.JwtVerification::handleRequest
```

Comment:
This path is wrong now, the folder that the handler class sits in now is jwtverification